### ISSUE TITLE
Failure to install blastr

### ISSUE THREAD

**Issue by incognitome** [gschofl/blastr#5]
Dear Gerhard,

I have R and Bioconductor updated:
Using Bioconductor 3.6 (BiocInstaller 1.28.0), R 3.4.1 (2017-06-30). 

I ran the following:
devtools::install_github("gschofl/blastr")

Here is the error message I got:

 --library='/usr/lib64/R/library' --install-tests 

* installing *source* package ‘blastr’ ...
** R
** inst
** tests
** preparing package for lazy loading
Error : object ‘fasta.info’ is not exported by 'namespace:Biostrings'
ERROR: lazy loading failed for package ‘blastr’
* removing ‘/usr/lib64/R/library/blastr’
Installation failed: Command failed (1)

Could you help? Thank you!

**Comment by Jakob37** [comment]
I encountered the exact same issue. Seems like a greatly useful tool otherwise!

**Comment by Jakob37** [comment]
I looked a bit closer. It seems like there are three exported functions that no longer are exported by the objects from which they are imported:

1. `fasta.info` is no longer exported from Biostrings. Here it is used to calculate FASTA-length. It seems to me like this could be replaced by `readBStringSet` from the Biostrings package.
2. There is an `unlist` function exported by IRanges. Is this needed, or could the built-in unlist be used?
3. `perl` regex is used. This is deprecated, and seems like it could be replaced by providing `perl` as argument to the desired function (which is done in some places).

After changing these the installation was successful. I will try using the package, but unsure if the changes might have influenced the functionality somehow - I am not confident enough in them to submit a pull request.

Seems like great work with this package btw, I hope it will be revived :)

**Comment by Jakob37** [comment]
Sorry about spamming here. I found another package which seems to have the same purpose. Might be worth giving a look: https://github.com/mhahsler/rBLAST

**Comment by incognitome** [comment]
No worries Jakob. I appreciate your sharing your findings. I ended up doing a lot of blasting manually... As for rBLAST, did you simply install it and was able to blast?

**Comment by Jakob37** [comment]
Yes, I think so. I executed it as following (I had already prepared by BLAST database running `makeblastdb` from the terminal, should be able to do that also through the package).

```
> blast_predictor <- rBLAST::blast(db="path_to_db.fasta", type="blastp")
> my_queries <- Biostrings::readDNAStringSet("path_to_seqs.fasta")
> blast_result <- predict(blast_predictor, my_queries)
> head(result)                                                         
            QueryID               SubjectID Perc.Ident Alignment.Length
1 Q4HYR2|ACEA_GIBZE    sp|P28297|ACEA_ARATH      32.32              526
2 Q4HYR2|ACEA_GIBZE    sp|O49290|CPPM_ARATH      35.66              129
3 Q4HYR2|ACEA_GIBZE   sp|Q9SII5|EXOL5_ARATH      32.61               46
4 Q4HYR2|ACEA_GIBZE   sp|Q9LL84|DNLI4_ARATH      32.14               56
5 Q4HYR2|ACEA_GIBZE sp|Q9ZVQ3-2|GSTZ1_ARATH      26.42               53
6 Q4HYR2|ACEA_GIBZE   sp|F4HRT5|CRWN1_ARATH      27.59               87
  Mismatches Gap.Openings Q.start Q.end S.start S.end       E Bits     
1        190           43      15   375      25   549 2.0e-22 98.6     
2         55            8      71   177      78   200 1.9e-02 36.2     
3         31            0     123   168     253   298 8.4e-01 30.8     
4         33            1     211   261     925   980 3.1e+00 29.6     
5         39            0       5    57       1    53 3.4e+00 28.9     
6         61            1     162   248     827   911 5.5e+00 28.5     
```

**Comment by virologist** [comment]
https://github.com/418704194/blastr
devtools::install_github("418704194/blastr")

![image](https://user-images.githubusercontent.com/54756814/161425662-8e45b8cc-b19b-458d-bd5c-e69216387a0d.png)
Is there anyone who has a tutorial?

**Comment by cjieming2** [comment]
> https://github.com/418704194/blastr devtools::install_github("418704194/blastr")
> 
> ![image](https://user-images.githubusercontent.com/54756814/161425662-8e45b8cc-b19b-458d-bd5c-e69216387a0d.png) Is there anyone who has a tutorial?

The above devtools::install_github("418704194/blastr") seems to work!
